Form-Controls/form-controls.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>T-shirt order</title>
+</head>
+<body>
+    <form>
+        <fieldset>
+            <legend>Customer Details</legend>
+
+            <label for="customer-name">Name</label>
+            <input type="text" name="name" id="customerName" minlength="2" placeholder="customer-name" required style="min-width:48px; min-height:48px;">>
+
+            <label for="customer-email">Email</label>
+            <input type="email" name="email" id="customerEmail" placeholder="customer-email" required style="min-width:48px; min-height:48px;">>
```

Comment:
The `placeholder` is visible to users of the website - is there a more human-readable placeholder we could use for our inputs?